source/tyr/tyr/resources.py

```diff
@@ -514,11 +527,18 @@ def post(self):
         if not billing_plan:
             return ({'error': 'billing plan doesn\'t exist'}, 400)
 
+        if args['shape']:
+            features =  ast.literal_eval(args['shape'])
```

Comment:
Same thing here, a `json.loads` should be enough.
Json look like python dict, but evaluating it as a python object is wrong